src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -110,15 +110,7 @@
                      (= page :status))
          (nav/navigate (:nav-context props) "workspaces"))
        [:div {}
-        [:div {:style {:padding "1em" :borderBottom (str "1px solid " (:line-gray style/colors))}}
-         [:div {:style {:float "right" :fontSize "70%"}}
-          [:span {:style {:marginRight "1ex" :color (:link-blue style/colors)}}
-           (-> (utils/get-current-user)
-               (utils/call-external-object-method :getBasicProfile)
-               (utils/call-external-object-method :getName))]
-          ;; Sadly, reloading the page seems to be the only reliable way to log out.
-          [:a {:href "javascript:;" :onClick (fn [e] (-> js/window .-location (.reload)))}
-           "Log-Out"]]
```

Comment:
Please add a story to follow up on this. From memory, I think a logout button is required for fisma, so we'll need to have that (which possibly requires orch-side changes too). Maybe a separate story for the name display; that's nice but not strictly necessary.